server/util/clickhouse/schema/schema.go

```diff
@@ -157,6 +157,10 @@ type Execution struct {
 
 	Stage int64
 
+	TargetID        string
```

Comment:
i know this is what the remote execution proto calls it, but ugh, i really don't like calling this an ID.  i guess this is non-actionable, just felt like i had to say it.

Reply:
Hmm that's a good point.

What do you think about `TargetName` or `ParentTarget`? I know that we do have a `Targets` table lurking around for the Target Tracker, so renaming this makes sense to me.

Reply:
The corresponding column in `Targets` is `Label`, so `TargetLabel` might be good if you want a suffix.